azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -115,6 +115,11 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"priority": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
could we add "ForceNew" here since a scale set cannot have a mix of Low and Regular VM. Unless Azure will update all pre-existing VM if you change your scaleset from `Regular` to `Low`?